pkg/sql/show.go

```diff
@@ -857,6 +860,191 @@ func (p *planner) ShowConstraints(
 	}, nil
 }
 
+func (p *planner) ShowQueries(ctx context.Context, n *parser.ShowQueries) (planNode, error) {
```

Comment:
Sorry, nevermind. Just found this note in the RFC: https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/monitoring_queries_and_sessions.md#a-note-on-permissions. That's actually the behavior I'm seeing.